bundles/framework/featuredata/publisher/FeaturedataTool.js

```diff
@@ -0,0 +1,71 @@
+Oskari.clazz.define('Oskari.mapframework.publisher.tool.FeaturedataTool',
+    function () {
+    }, {
+        index: 9,
+        // Disabled for now, need to fix config reading first allowedLocations: ['top left', 'top right', 'bottom left', 'bottom right'],
+        lefthanded: 'top right',
+        righthanded: 'top right',
+        /**
+        * Get tool object.
+        * @method getTool
+        * @private
+        *
+        * @returns {Object} tool
+        */
+        getTool: function () {
+            return {
+                id: 'Oskari.mapframework.bundle.featuredata.plugin.FeaturedataPlugin',
+                title: 'FeaturedataPlugin',
+                config: this.state.pluginConfig || {}
+            };
+        },
+        // Key in view config non-map-module-plugin tools (for returning the state when modifying an existing published map).
+        bundleName: 'featuredata',
+
+        /**
+         * Initialise tool
+         * @method init
+         */
+        init: function (data) {
+            console.log('No voihan flöff.');
```

Comment:
flöff sentään :P

Reply:
oopsies. Rebase to the rescue - the console log was never there ;)